### ISSUE TITLE
unsolved definition (e.g "a := a", "a, b := a, b") doesn't give error

### ISSUE THREAD

**Issue by SleepyRoy** [vlang/v#4981]
**V version:** 
latest master with mingw64

**OS:** 
win10

**What did you do?**
```V
fn main() {
	a, b := a, b
}
```

**What did you expect to see?**
an error
 
**What did you see instead?**
warnings: a and b are unused
then v.exe crashed

**Comment by SleepyRoy** [comment]
This has the same issue.
```V
fn main() {
	a := a
}
```
Strange, I remember this used to give unsolved error...

**Comment by buttercrab** [comment]
```v
fn main() {
	a := b
	b := a
}
```

Maybe related error code. It crashes. (Same with chain of three variables)

**Comment by spytheman** [comment]
@buttercrab your example is fixed in the next commit: 23e8c8e
